examples/stable_diffusion/server.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 MOSEC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import torch  # ingore: type
+from diffusers import StableDiffusionPipeline  # ignore: type
+
+from mosec import Server, Worker
+from mosec.mixin import MsgpackMixin
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    "%(asctime)s - %(process)d - %(levelname)s - %(filename)s:%(lineno)s - %(message)s"
+)
+sh = logging.StreamHandler()
+sh.setFormatter(formatter)
+logger.addHandler(sh)
+
+
+def StableDiffusion(MsgpackMixin, Worker):
+    def __init__(self):
+        self.pipe = StableDiffusionPipeline.from_pretrained(
+            "runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16
+        )
+        self.pipe = self.pipe.to("cuda")
+
+    def forward(self, data):
```

Comment:
what's the type of `data`?

Reply:
Should be a list of strings. Will update the type hint.